README.md

```diff
@@ -6,12 +6,18 @@
 
 ## Initialize
 
-In order to start using spk, you need to specify configuration in a yaml file.
-Please refer to the config file located [here](./spk-config.yaml) and follow the
-same format.
+To start using spk, specify its configuration in a `yaml` file. Refer to the
+[spk-config.yaml](./spk-config.yaml) file and follow the same format.
 
-If you would like to specify private keys or access tokens that should **not be
-stored in raw text**, follow the steps below.
+### Environment Variables
+
+To specify private keys or access tokens that should **not be stored in raw
```

Comment:
I would add a link to `.env.example` file and explain all environment variables in the example file are expected to be configured by the user.